Source/Csla/Core/BusinessBase.cs

```diff
@@ -1159,7 +1160,9 @@ void IHostRules.AllRulesComplete()
     /// </remarks>
     protected virtual void AddBusinessRules()
     {
-      BusinessRules.AddDataAnnotations();
+      var options = ApplicationContext.GetRequiredService<CslaOptions>();
```

Comment:
What about doing this instead of here there: https://github.com/MarimerLLC/csla/blob/faa3c2d973cf7b9e21f864b45df0b1b07d6d782d/Source/Csla/Rules/BusinessRules.cs#L1167 ?